scrabble_cheater.py

```diff
@@ -0,0 +1,767 @@
+import re
+from random import randrange
+from scrabble_constants import WWF_LET_VALUES, WWF_BONUS_TILES, BONUS_TILE_COLORS
+import sys
+import random
+import Tkinter as tk
+
+BOARD_SIZE = 15
+BLANK_TILE = "_"
+occupied_squares = []
+scrabble_board = [[None for x in range(BOARD_SIZE)] for y in range(BOARD_SIZE)]
+dictionary = {}
+WORD_FILE = "ospd.txt"
+
+# Conventions:
+# Specific vectors (rows or columns) are referenced with strings of the form
```

Comment:
I dont really like this convention. it's lightweight sure but seems like a class would be better suited for this.